parm/atm/obs/testing/satwind.yaml

```diff
@@ -847,25 +870,140 @@ obs post filters:
+        name: ObsFunction/ObsErrorFactorDuplicateCheck
+        options:
+          use_air_pressure: true
+          variable: windEastward
+  - filter: BlackList
+    filter variables:
+    - name: windNorthward
+    action:
+      name: inflate error
+      inflation variable:
+        name: ObsFunction/ObsErrorFactorDuplicateCheck
+        options:
+          use_air_pressure: true
+          variable: windNorthward
+  # We are extending this to an additional filter that inflates final ob-errors across-the-board by
+  # 1/0.8 = 1.25. This is caused by the GSI value of nvqc being set to .true. in the global operational
+  # configuration, see: https://github.com/NOAA-EMC/global-workflow/blob/d5ae3328fa4041b177357b1133f6b92e81c859d7/scripts/exglobal_atmos_analysis.sh#L750
+  # This setting activates Line 1229 of setupw.f90 to scale ratio_errors by 0.8, which is applied in
+  # the denominator of the final ob-error, so 1/0.8 = 1.25 factor of ob-error inflation.
   #
-  # All satwinds subject to a SPDB check (function may be broken?)
-  #   We are ignoring this filter for now, and will come back to
-  #   the issue of how to handle this SPDB check later. For now,
-  #   observations that are GSI-rejected based on this test are
-  #   not being considered when checking acceptance compliance.
-  #- filter: Bounds Check
-  #  filter variables:
-  #  - name: windEastward
-  #  - name: windNorthward
-  #  test variables:
-  #  - name: ObsFunction/SatWindsSPDBCheck
-  #    options:
-  #      error_min: 1.4
-  #      error_max: 20.0
-  #  maxvalue: 1.75
-  #  action:
-  #    name: reject
-  #  defer to post: true
+  # If this nvqc functionality were to be switched off (i.e. if variational qc were to be turned off),
+  # you would want to remove this last inflation filter.
+  - filter: Perform Action
+    filter variables:
+    - name: windEastward
+    where:
+    - variable: ObsType/windEastward
+      is_in: 240-260
+    action:
+      name: inflate error
+      inflation factor: 1.25
+#
+  - filter: Perform Action
+    filter variables:
+    - name: windNorthward
+    where:
+    - variable: ObsType/windNorthward
+      is_in: 240-260
+    action:
+      name: inflate error
+      inflation factor: 1.25
+# END OF FILTERS
 linear obs operator:
   name: Identity
 
+passedBenchmark: 1025814 # 2 variables (u,v), both passing 512907 obs, including 512907 GSI/UFO agreements, and:
+                         # 16 GOES AMVs (6 u-obs, 6 v-obs) that are rejected by the SatWindsErrnormCheck in UFO
+                         # but are retained in GSI's equivalent experr_norm check. All 6 of these disagreements
+                         # have error norm values at almost exactly 0.9, indicating a float precision/handling
+                         # difference between UFO and GSI generating the disagreement.
+                         #
+                         # NOTE: This benchmark is specifically for a test with assim_freq=6, which based on
```

Comment:
I love this level of detail in the comment here, this is a great reference for us!

Reply:
I agree, this might be particularly useful to have with the conventional YAMLs that we are yet to add.